cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -1095,6 +1095,7 @@ func NewUpstreamController() (*UpstreamController, error) {
 	ml, err := messagelayer.NewContextMessageLayer()
```

Comment:
NewContextMessageLayer() function only returns an `ContextMessageLayer object`  and the err always is nil.
I suggest modifying the return value of this function。

`func NewContextMessageLayer() (MessageLayer, error)` -> `func NewContextMessageLayer() (MessageLayer)`

Reply:
Sounds good.

Reply:
agreed, updated.